### DISCUSSION TITLE
Problema de Ingreso en Microsoft Teams

### DISCUSSION THREAD

**Discussion by devjesusml** [Launch-X-Latam/MisionFrontEnd#482] [Team Posts]
Gusto en saludarles,
No puedo acceder al canal de Microsoft Teams, debido a que en el Forms me aparece que no tengo acceso para contestarlo. Si me pudieran ayudar con esto, se les agradecería muchísimo.

**Comment by Gerardo-Guizar** [comment]
Hola, intenta hacerlo en una sesión de incognito o limpia las cookies porque pueden estar ocasionando ese problema, a mi me pasó lo mismo y así lo resolví. Saludos
